atmospheric_order_download/cda_download.py

```diff
@@ -871,6 +873,14 @@ def backoff_time_calculator(count, limit):
         default=True,
         help="Turn off the verification of SSL on requests.get.",
     )
+    parser.add_argument(
+        "-g",
+        "--fillgaps",
+        action="store",
+        dest="fillgaps",
+        default=True,
+        help="Optional: Only download the gaps in files if selected"
+    )
```

Comment:
One thing I noticed is that if you already have an existing destination folder without the date in the folder name, then fillGaps will skip all the files, because they already exist for a different run. So it won't overwrite the previous day's data. 

If we're going to default fillGaps to True we should also default folderDate to True as well, so that a unique folder will exist for each run.